pymeasure/instruments/advantest/advantestR3767CG.py

```diff
@@ -67,10 +67,10 @@ class AdvantestR3767CG(Instrument):
         "TRAC:DATA? FDAT1", """ Reads the Data array from trace 1 after formatting """
     )
 
-    def __init__(self, resourceName, **kwargs):
+    def __init__(self, adapter, **kwargs):
```

Comment:
Since we are renaming posargs in a large number of instrument implementations, we should add an entry to a fresh section (upcoming release) in the CHANGES.rst where we make note of this. 
I suspect we could break user code if it refers to the posarg by name, but the name changed? (I.e. `myinstr = AdvantestR3767CG(resourceName='COM2')`.)